bin/templates/scripts/cordova/Api.js

```diff
@@ -202,7 +217,7 @@ Api.prototype.addPlugin = function (plugin, installOptions) {
  */
 Api.prototype.removePlugin = function (plugin, uninstallOptions) {
     var Plugman = require('./lib/plugman/Plugman');
-    return Plugman.get(this.locations, this.events).removePlugin(plugin, uninstallOptions);
+    return Plugman.get(this.locations, events).removePlugin(plugin, uninstallOptions);
```

Comment:
Looks like `Plugman` doesn't use `events` parameter. Can we just remove it here and in `Plugman`?